source/js/components/join/join.scss

```diff
@@ -2,27 +2,68 @@
   background: inherit;
 
   .has-danger {
-    color: $red;
+    color: $dark-red;
+
+    .form-control {
+      border: 2px solid $dark-red;
+    }
+
+    .glyph-container {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      right: 10px;
+      height: 13px;
+      width: 13px;
+      margin: auto;
+    }
+
+    .form-error-glyph:nth-child(2) {
```

Comment:
Let's clean up how we set `margin-left` for this element. Currently I can see it being set three times but I can't understand why. Also I would suggest giving this targeted `.form-error-glyph` element a class name so we can just target it using that class name instead of `.form-error-glyph:nth-child(2)`.

<img width="744" alt="image" src="https://user-images.githubusercontent.com/2896608/64413638-b6857b00-d046-11e9-817e-ecd1872f6936.png">